src/measure_theory/measure_space.lean

```diff
@@ -1600,13 +1600,20 @@ open measure
 
 /-- A measure `μ` is called σ-finite if there is a countable collection of sets
   `{ A i | i ∈ ℕ }` such that `μ (A i) < ⊤` and `⋃ i, A i = s`. -/
-@[class] def sigma_finite (μ : measure α) : Prop :=
-nonempty (μ.finite_spanning_sets_in {s | measurable_set s})
+class sigma_finite (μ : measure α) : Prop :=
+(out' : nonempty (μ.finite_spanning_sets_in {s | measurable_set s}))
```

Comment:
The nonempty is redundant now, right?

Reply:
No, you can't use `class` to declare `nonempty` or `exists`-like inductives, because lean wants to be able to construct the projections.

Reply:
Of course, you're right.  That's why we had `class inductive`.